app/views/content/life-as-a-teacher/change-careers/how-to-change-careers-to-become-a-teacher/_article.html.erb

```diff
@@ -0,0 +1,33 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+  <h2 class="heading--box-blue">What you'll need to become a teacher</h2>
+    <p>You’ll need a degree and <a href="/train-to-be-a-teacher/what-is-qts">qualified teacher status (QTS)</a> to work in the majority of schools in England. This  includes state maintained primary, secondary and special schools.</p>
+    <p>There are different ways to get QTS.</p>
+    <h3>If you have a degree</h3>
+    <p>If you already have a degree, you can do a postgraduate teacher training course to get QTS.</p>
+    <p>It does not matter when you graduated, and your degree subject does not necessarily need to be in the subject you want to teach. Once you have QTS, you can also teach any other subject, as long as you can show you have enough expertise.</p>
+    <p>Most postgraduate teacher training courses charge fees. There are ways to fund your training, for example, through tuition fee and maintenance loans. You may also be eligible for a bursary or scholarship of up to <%= v :scholarships_generic_maxshortened %>.</p>
+    <p>Other postgraduate teacher training courses come with a salary. This means you will not pay tuition fees and will receive a salary while you train. These courses are in high demand and very competitive, so it’s important to apply as soon as you can if you’re eligible.</p>
```

Comment:
I wonder if it is worth adding this line (from salaried page):

`If you’re eligible for a bursary or scholarship, you could receive more money on a non-salaried course than on a salaried one.`